### PR TITLE
ensure Suricata events rules are added back after switching to Suricata from Snort

### PR DESCRIPTION
**Pull request by weslambert**

### CODE REVIEW

File: usr/bin/rule-update [hunk 1]
```diff
@@ -176,6 +176,13 @@ if [ ! -f $SSH_CONF ]; then
 
 	# Suricata options
 	if [ "$ENGINE" = "suricata" ]; then
+		# Check for local Snort rules that need to be removed
+		if grep -x "^local_rules=/etc/nsm/rules/local.rules" /etc/nsm/pulledpork/pulledpork.conf >/dev/null 2>&1; then
+			# Backup pulledpork.conf
+			cp /etc/nsm/pulledpork/pulledpork.conf /etc/nsm/pulledpork/pulledpork.conf.bak.`date +%Y%m%d`
+			# Remove local Snort rules
```
Comment (dougburks) [inline usr/bin/rule-update]: Is this comment correct?

File: usr/bin/rule-update [hunk 2]
```diff
@@ -176,6 +176,13 @@ if [ ! -f $SSH_CONF ]; then
 
 	# Suricata options
 	if [ "$ENGINE" = "suricata" ]; then
+		# Check for local Snort rules that need to be removed
```
Comment (dougburks) [inline usr/bin/rule-update]: Is this comment correct?

**Review by dougburks — CHANGES_REQUESTED** [review]
Hi Wes,

As always, thanks for the pull request!

Please see comments.

**Comment by weslambert** [comment]
I've revised the comments to be a little more accurate.

**Pull request merged**